docs/development.rst

```diff
@@ -23,16 +26,30 @@ Quickstart
 
   $ git submodule update --init --recursive
 
+  (note that on Windows you may have to run this as administrator, in order to create the
+  correct symlinks)
 - Install the Python development requirements using
   ``pip install -r python/requirements/development.txt``.
 - Build the low level module by running ``make`` in the ``python`` directory.
 - Run the tests to ensure everything has worked: ``python -m nose -vs``. These should
   all pass.
+
+Adding features
+---------------
+
 - Make your changes in a local branch, and open a pull request on GitHub when you
   are ready. Please make sure that (a) the tests pass before you open the PR; and
   (b) your code passes PEP8 checks (see below for a git commit hook to ensure this
   happens automatically) before opening the PR.
 
+Installing your local version as the python default
+---------------------------------------------------
+
+- If you (optionally) want to install your local ``tskit`` version as the default Python
+  version, point pip to the ``python`` directory, where ``setup.py`` resides. E.g. at the
```

Comment:
This is confusing to novices --- what does pointing pip at something mean? Also, what does pip -e do?

Reply:
How about 

```
- If you (optionally) want to install your local ``tskit`` version as the default Python
  version, you should be able to do ``python3 -m pip install -e python`` (the ``-e`` flag
  tells pip to install from the local tskit ``python`` directory).
```